mqttasgi/server.py

```diff
@@ -203,7 +203,7 @@ async def mqtt_subscribe(self, app_id, msg):
                 "[mqttasgi][app][subscribe] - Subscription to {} must be updated to QOS: {}".format(topic, qos))
             self.client.unsubscribe(topic)
             self.client.subscribe(topic, qos)
-            status = (qos, status[1])
+            status = qos
```

Comment:
As @mzitelli97 mentioned, is the right line. please ammend
`status['qos'] = qos`